packages/kbn-optimizer/limits.yml

```diff
@@ -106,4 +106,4 @@ pageLoadAssetSize:
   stackAlerts: 29684
   presentationUtil: 28545
   spacesOss: 18817
-  indexPatternFieldEditor: 23489
+  indexPatternFieldEditor: 90489
```

Comment:
Here ~70Kb is added to `indexPatternFieldEditor` because of the moved code, maybe it is possible to subtract ~70Kb from the index pattern management plugin.